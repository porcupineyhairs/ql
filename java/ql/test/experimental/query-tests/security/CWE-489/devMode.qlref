experimental/Security/CWE/CWE-489/devMode.ql
