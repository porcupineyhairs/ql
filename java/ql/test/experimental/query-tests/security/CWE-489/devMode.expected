| StrutsBad.xml:8:5:8:52 | constant |
